app/models/tv_credit_season.rb

```diff
@@ -0,0 +1,34 @@
+class TVCreditSeason
+  attr_accessor :air_date, :episode_count, :id, :name, :overview, :poster_path, :season_number, :show_id
+
+  def initialize(air_date:, episode_count:, id:, name:, overview:, poster_path:, season_number:, show_id:)
+    @air_date = air_date
+    @episode_count = episode_count
+    @id = id
+    @name = name
+    @overview = overview
+    @poster_path = poster_path
+    @season_number = season_number
+    @show_id = show_id
+  end
+
+  def self.parse_records(json)
+    json.map do |record|
+      parse_record(record)
+    end.sort_by { |r| r.season_number }
+  end
+
+  def self.parse_record(record)
```

Comment:
I've broken the parsing into 2 different methods. This allows the caller to choose whether they're trying to parse a single object or an array of these objects. I chose the word `record` because it is easy to tell when it is single/plural.